package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mean-expert/loopback-sdk-builder",
-  "version": "2.1.0-rc.9",
+  "version": "2.1.0-rc.10.2",
```

Comment:
Why is this change part of this pull request? it seems it squashed commits non related to this functionality.. This modifies the history